File: app/src/main/java/com/druger/refresher/model/ModelSeparator.java
package com.druger.refresher.model;

import com.druger.refresher.R;

/**
 * Created by druger on 30.09.2015.
 */
public class ModelSeparator implements Item {

    public static final int TYPE_OVERDUE = R.string.separator_overdue;
    public static final int TYPE_TODAY = R.string.separator_today;
    public static final int TYPE_TOMORROW = R.string.separator_tomorrow;
    public static final int TYPE_FUTURE = R.string.separator_future;

    private int type;

    public ModelSeparator(int type) {
        this.type = type;
    }

    public int getType() {
        return type;
    }

    public void setType(int type) {
        this.type = type;
    }

    @Override
    public boolean isTask() {
        return false;
    }
}

File: app/src/main/java/com/druger/refresher/models/ModelSeparator.kt
package com.druger.refresher.models;

import com.druger.refresher.R;

/**
 * Created by druger on 30.09.2015.
 */
class ModelSeparator(val type: Int) : Item {

    companion object {
        const val TYPE_OVERDUE = R.string.separator_overdue;
        const val TYPE_TODAY = R.string.separator_today;
        const val TYPE_TOMORROW =  R.string.separator_tomorrow;
        const val TYPE_FUTURE = R.string.separator_future;
    }

    override fun isTask(): Boolean {
        return false;
    }
}
